java/client/src/main/java/glide/api/models/commands/SetOptions.java

```diff
@@ -107,11 +104,26 @@ public SetOptionsBuilder conditionalSetOnlyIfNotExist() {
          * <p>This method overrides any previously set {@code conditionalSet} and {@code
          * comparisonValue}.
          *
-         * @since Valkey 8.1 and above.
          * @param value the value to compare
          * @return this builder instance
+         * @since Valkey 8.1 and above.
          */
         public SetOptionsBuilder conditionalSetOnlyIfEqualTo(@NonNull String value) {
+            return conditionalSetOnlyIfEqualTo(GlideString.gs(value));
+        }
+
+        /**
+         * Sets the condition to {@link ConditionalSet#ONLY_IF_EQUAL} for setting the value. The key
+         * will be set if the provided comparison value matches the existing value.
+         *
+         * <p>This method overrides any previously set {@code conditionalSet} and {@code
+         * comparisonValue}.
+         *
+         * @param value the value to compare
+         * @return this builder instance
+         * @since Valkey 8.1 and above.
+         */
```

Comment:
Would it be better to use `@inheritDoc` so the documentation stays consistent?


```suggestion
        /**
         * {@inheritDoc}
         */
```